kogito-editors-js/packages/boxed-expression-component/src/components/BoxedExpressionEditor/BoxedExpressionProvider.css

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-.boxed-expression-editor {
```

Comment:
I think this change will break some parts of `kogito-tooling`

### Code
- https://github.com/kiegroup/kogito-tooling/blob/main/packages/unitables/src/dmn/style.css#L15

### Tests
- https://github.com/kiegroup/kogito-tooling/blob/main/packages/vscode-extension-pack-kogito-kie-editors/it-tests/helpers/dmn/ExpressionEditorHelper.ts#L50